---
layout: work
title: Skills and Proficiencies
permalink: /skills
order: 2
items:
  - title: Clustering
    image:
      src: /assets/img/cluster.png
      alt: cluster TSNE
    description: k-means clustering, graph-based clustering (Leiden, Louvain), spatially aware hidden Markov random field clustering models (BayesSpace, PRECAST)
  - title: Integration
    image:
      src: /assets/img/dna.png
      alt: DNA helix
    description: Batches (Seurat, MNN, Harmony, scVI), Multi-omic (Seurat/Signac, ArchR), Multi-modal (NMF), Genomic (PRS, LDSC, MAGMA, eQTL)
  - title: Differential expression models
    image:
      src: /assets/img/volcano.png
      alt: volcano plot
    description: Pseudobulk (limma/ edgeR (eBayes), DESeq2 (apeglm shrinkage)), Single-cell (MAST, Wilcoxon), ATAC differential accessibility
  - title: Gene sets and networks
    image: 
      src: /assets/img/gene-net.png
      alt: gene netword
    description: GSEA (fgsea), ORA (clusterProfiler, EnrichR), hdWGCNA, scGREAT, SCENIC
---

<p>
  The methodology implemented for any NGS project depends on the data modality, the project design, and the hypotheses being tested. For the past five years I have worked extensively with RNA-sequencing data (single-cell, single-nuclei, spatial, long-read) and multiomic sequencing data (single-nuclei RNA-sequencing and ATAC-sequencing). Here, I summarize methods that highlight my expertise with NGS analysis, gathered over many different projects. 
</p>
<p>
  Working with NGS data requires extensive organization and documentation that are often implemented in high-performance computing environments and achieved with tools like Github for version control and conda or pip package management. My experience with different sequencing techniques has given me the skills to be comfortable processing NGS data from fastq files, manually performing quality control, barcode extraction, read alignment, and read quantification. Across the years I have used R and Python extensively, executing code with bash scripts, Jupyter Notebook, and integrated development environments like RStudio and PyCharm. I have worked with all of the major NGS processing pipelines including BioConductor, Seurat, and scanpy workflows.
</p>
