---
layout: work
title: Skills and proficiencies
permalink: /skills
order: 2
items:
  - title: Clustering
    image:
      src: /assets/img/cluster.png
      alt: cluster TSNE
    description: k-means clustering, graph-based clustering (Leiden, Louvain), spatially aware hidden Markov random field clustering models (BayesSpace, PRECAST)
  - title: Integration
    image:
      src: /assets/img/dna.png
      alt: DNA helix
    description: Batches (Seurat, MNN, Harmony, scVI), Multi-omic (Seurat/Signac, ArchR), Multi-modal (NMF), Genomic (PRS, LDSC, MAGMA, eQTL)
  - title: Differential expression models
    image:
      src: /assets/img/volcano.png
      alt: volcano plot
    description: Pseudobulk (limma/ edgeR (eBayes), DESeq2 (apeglm shrinkage)), Single-cell (MAST, Wilcoxon), ATAC differential accessibility
  - title: Gene sets and networks
    image: 
      src: /assets/img/gene-net.png
      alt: gene netword
    description: GSEA (fgsea), ORA (clusterProfiler, EnrichR), hdWGCNA, scGREAT, SCENIC
---

This is an example of a "Work" page, displaying your work, your interests, your projects.
<br />
Data modalities: 10X spatial transcriptomics (3â€™ Visium (55um), VisiumHD), 10X single-cell and single-nuclei RNAseq, 10X multiomic snRNAseq and snATACseq, Oxford Nanopore long-read RNAseq
<br />
Standard best practices operations: Slurm high-performance computing environment, Git version history control Conda/ pip environment controls
<br />
Command-line tools: Tools for long read (bamtools, etc), ATAC tools (MACS2, etc), 10X cellranger and spaceranger.
<br />
Languages: R (Bioconductor workflow, Seurat workflow), Python (Scanpy workflow, PyCharm/ Tensorflow)
<br />
<br />
