---
layout: page
title: Skills and Proficiencies
permalink: /skills/
order: 2
---

<h3> Data modalities </h3>

- 10X 3â€™ Visium (55um)
- 10X VisiumHD (some experience)
- 10X single-cell RNAseq
- 10X single-nuclei RNAseq
- 10X multiomic snRNAseq and snATACseq
- Oxford Nanopore cDNA long-read sequencing

<h3> Languages/ Platforms </h3>

- Standard best practices operations
  - Slurm high-performance computing environment,
  - Git version history control
  - Conda/ pip environment controls
- Command-line
  - Tools for long read (bamtools, etc)
  - ATAC tools (MACS2, etc)
  - 10X cellranger and spaceranger
- R
  - Bioconductor workflow
  - Seurat workflow
- Python
  - Scanpy workflow
  - PyCharm/ Tensorflow
 
<h3> Data processing </h3>

- Clustering
  - k-means clustering, graph-based clustering (Leiden, Louvain), spatially aware hidden Markov random field clustering models (BayesSpace, PRECAST)
- Differential expression models
  - Pseudobulk: limma/ edgeR (eBayes), DESeq2 (apeglm shrinkage)
  - Single-cell: MAST, Wilcoxon
  - Differential accessibility (ATAC)
- Integration
  - Batches: Seurat, MNN, Harmony, scVI
  - Multi-omic: Seurat/Signac, ArchR
  - Multi-modal: NMF
  - Genomic: PRS, LDSC, MAGMA
- Gene set analysis
  - GSEA (fgsea), ORA (clusterProfiler, EnrichR)
- Gene regulatory networks
  - hdWGCNA, scGREAT, SCENIC



